static/app/views/discover/table/columnEditModal.spec.tsx

```diff
@@ -632,7 +632,7 @@ describe('Discover -> ColumnEditModal', function () {
   });
 
   describe('equation automatic update', function () {
-    let onApply;
+    let onApply!: jest.Mock;
```

Comment:
Is there any other way that allows us to not use `!`? 🤔

Reply:
we'll need to use it even more if we keep making the types more strict since typescript doesn't know how beforeEach works